tests/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -0,0 +1,68 @@
+import { mount } from 'enzyme';
+import mockManager from '../../../setup/managermocker';
+import DOM from '../../../../src/ui/dom/dom';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import AlternativeVerticalsComponent from '../../../../src/ui/components/results/alternativeverticalscomponent';
+
+DOM.setup(document, new DOMParser());
+let COMPONENT_MANAGER, defaultConfig, storage;
+
+beforeEach(() => {
+  COMPONENT_MANAGER = mockManager();
+  COMPONENT_MANAGER.getComponentNamesForComponentTypes = () => [];
+  const bodyEl = DOM.query('body');
+  DOM.empty(bodyEl);
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+  storage = COMPONENT_MANAGER.core.storage;
+  delete global.window.location;
+  global.window.location = {
+    search: '?query=office+space'
+  };
+
+  defaultConfig = {
+    container: '#test-component',
+    verticalsConfig: [
+      {
+        label: 'Home',
+        url: 'index.html'
+      },
+      {
+        verticalKey: 'KM',
+        label: 'Locations',
+        url: 'locations.html'
+      }
+    ],
+    baseUniversalUrl: './index.html?query=office+space',
+    data: {
+      alternativeVerticals: [{
+        resultsCount: 2,
+        verticalConfigId: 'KM'
+      }]
+    }
+  };
+});
+
+describe('AlternativeVerticals component', () => {
+  it('uses query from the url in suggestLinks', () => {
```

Comment:
This test does a bit more than make sure that the query is in the `suggestLink`. It basically makes sure that the entire `suggestLink` is created properly. Could we rename the test `constructs vertical suggest link properly`?